### DISCUSSION TITLE
Inappropriate, even racial content generated by Copilot

### DISCUSSION THREAD

**Discussion by EdiWang** [community/community#106376] [Copilot Conversations] [answered]
### Select Topic Area

Product Feedback

### Body

Copilot can generate inappropriate content. Please consider adding a content safety filter.

### Racial

![image](https://github.com/community/community/assets/3304703/b419877f-8fe2-46d1-985f-4e65a321663a)

![image](https://github.com/community/community/assets/3304703/ab4248e2-3d00-4c0d-8155-52092dbd7e2c)

![image](https://github.com/community/community/assets/3304703/f4ce1901-631b-4b93-889d-468af3c07bea)

![image](https://github.com/community/community/assets/3304703/5945e68d-d24b-4f53-bff7-5c8038e16d21)

### Violent

![image](https://github.com/community/community/assets/3304703/724b24f1-3dc4-4bb4-a44f-83e2ccb647a1)

![image](https://github.com/community/community/assets/3304703/f2197c57-aaab-4044-b9b2-36aee244f5b3)

**Comment by davecheney** [comment, marked as answer]
@EdiWang i'm very sorry that you had to experience this. Please contact support; https://support.github.com/contact who will escalate to the appropriate teams internally.
